apps/web/src/pages/activities/components/ActivityItem.tsx

```diff
@@ -6,19 +6,39 @@ import { format } from 'date-fns';
 import { useNotificationStatus } from '../hooks/useNotificationStatus';
 import { ActivityStep } from './ActivityStep';
 import { CheckCircle, ErrorIcon, Timer } from '../../../design-system/icons';
-import { ExecutionDetailsStatusEnum, StepTypeEnum } from '@novu/shared';
+import { ExecutionDetailsStatusEnum } from '@novu/shared';
 import { useEffect, useState } from 'react';
+import { DigestedStep } from './DigestedStep';
 
 const JOB_LENGTH_UPPER_THRESHOLD = 3;
-const checkJobsLength = (item) => item.jobs.length > JOB_LENGTH_UPPER_THRESHOLD;
+const checkJobsLength = (item) => {
+  let length = item.jobs.length;
+  if (item._digestedNotificationId) {
+    length = length + 1;
+  }
```

Comment:
```suggestion
  const length = getJobsLength(item);
```
Super clean with the new method created.

Reply:
So right🤩